app/src/main/java/com/mapbox/maps/testapp/examples/LegacyOfflineActivity.kt

```diff
@@ -56,7 +56,7 @@ class LegacyOfflineActivity : AppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     setContentView(R.layout.activity_offline)
-    offlineManager = OfflineRegionManager(MapboxOptions.getDefaultResourceOptions((this)))
+    offlineManager = OfflineRegionManager(MapInitOptions.getDefaultResourceOptions(this))
```

Comment:
would it make sense to move the `getDefaultResourceOptions(context)` to CredentialsManager?

Reply:
cc @julianrex since this is still not aligned with iOS.

Reply:
It sounds reasonable to me, but need to align with iOS on this.

Reply:
Just checked the [codes](https://github.com/mapbox/mapbox-maps-ios/pull/254/files#diff-46cda2fdea49d4ea82517a0b932c27ede2548397ba7e735ecc6798c44095b130R27) for iOS, it seems there is no such API

Reply:
we had similar conversation [here](https://github.com/mapbox/mapbox-maps-ios/pull/254/files#r611443274), let's discuss and create follow-up PR for iOS cc @julianrex